tools/shell-commands/src/main/java/org/apache/unomi/shell/migration/utils/MigrationUtils.java

```diff
@@ -216,7 +216,9 @@ public static void reIndex(CloseableHttpClient httpClient, BundleContext bundleC
             // Recreate the original index with new mappings
             HttpUtils.executePutRequest(httpClient, esAddress + "/" + indexName, newIndexSettings, null);
             // Reindex data from clone
-            HttpUtils.executePostRequest(httpClient, esAddress + "/_reindex", reIndexRequest, null);
+            JSONObject task = new JSONObject(HttpUtils.executePostRequest(httpClient, esAddress + "/_reindex?wait_for_completion=false", reIndexRequest, null));
+            //Wait for the reindex task to finish
+            waitForTaskToFinish(httpClient, esAddress, task.getString("task"), migrationContext);
```

Comment:
A reindex query is also called in the method MigrationUtils.moveToIndex, this one might also need to be modified like it's done here

Reply:
Fixed